apps/api/src/controllers/admin.schema.ts

```diff
@@ -85,6 +85,14 @@ export const AdminStartCompetitionSchema = z
     endDate: z.iso.datetime().optional(),
     votingStartDate: z.iso.datetime().optional(),
     votingEndDate: z.iso.datetime().optional(),
+    tradingConstraints: z
```

Comment:
@joewagner lookin good. one thing we should do is alter the [`UpdateCompetitionSchema`](https://github.com/recallnet/js-recall/blob/ebe20a1060d8ceca92f3fb06b4388d222cc23437/apps/api/src/database/schema/core/types.ts#L18) in case we need to hotfix this for whatever reason.

the current `UpdateCompetitionSchema` shape only accounts for the `competitions` schema but doesnt include `tradingConstraints`. we should also add `tradingConstraints` to the `AdminCreateCompetitionSchema` schema